util.go

```diff
@@ -15,3 +16,10 @@ func uuid() string {
 	id[8] |= 0x80 // set to IETF variant
 	return hex.EncodeToString(id)
 }
+
+func checkFileExist(fileName string) bool {
```

Comment:
Can we name it `fileExists` instead? It reads better in conditionals

```go
if fileExists('some/path.go') {
  // do thing
}
```

Reply:
Makes sense.